community/knative/charts/serving/charts/monitoring/Chart.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: v1
+appVersion: "0.2.3"
+description: A Helm chart for Kubernetes
+name: monitoring
+version: 0.1.0
+icon: https://raw.githubusercontent.com/knative/docs/master/images/logo/rgb/knative-logo-rgb.jpg
+kubeVersion: ">=1.10.3"
+tillerVersion: ">=2.10.0"
```

```suggestion
tillerVersion: ">=2.9.1"
```
Needs to support at least 2.9.1 for ICP 3.1.x support.

---

same as [#184 (comment)](https://github.com/IBM/charts/pull/184#discussion_r255170249)